### ISSUE TITLE
Quick Diff selection dropdown is not keyboard accessible

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#174926]
Testing #174853

The quick diff dropdown widget cannot be accessed via keyboard

**Comment by alexr00** [comment]
Thanks for catching this!
